Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -144,6 +156,13 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
             collectionView.contentInset.bottom += typingIndicatorViewHeight
         }
         
+        view.addSubview(scrollToBottomButton)
+        scrollToBottomButton.trailingAnchor.pin(equalTo: view.layoutMarginsGuide.trailingAnchor).isActive = true
+        scrollToBottomButton.bottomAnchor.pin(equalTo: collectionView.bottomAnchor, constant: -8).isActive = true
```

Comment:
You can use one of the `greaterThanOrEqualToSystemSpacing***` methods instead of hardcoding `8`